packages/jellyfish-api-core/src/category/poolpair.ts

```diff
@@ -162,17 +162,17 @@ export class PoolPair {
   /**
    * Remove pool liquidity transaction
    *
-   * @param {string} address defi address for crediting tokens
-   * @param {string} poolAccount pool liquidity account of owner
+   * @param {string} address defi address containing the liquidity tokens
+   * @param {string} amount 'liquidityAmount@liquidity' pool symbol, e.g. BTC-DFI
    * @param {PoolLiquidityOptions} [options]
    * @param {UTXO[]} [options.utxos] utxos array of specific UTXOs to spend
    * @param {string} [options.utxos.txid]
    * @param {number} [options.utxos.vout]
    * @return {Promise<string>}
    */
-  async removePoolLiquidity (address: string, poolAccount: string, options: PoolLiquidityOptions = {}): Promise<string> {
+  async removePoolLiquidity (address: string, amount: string, options: PoolLiquidityOptions = {}): Promise<string> {
```

Comment:
I prefer the old naming, `123@BTC-DFI` is not an "amount".

If we were going to overhaul things, I will go for amount and pair ID as 2 individual argument, but not likely for sake of backward compatibility.

Reply:
Got it. I think we don't need to seperate things as well. I'll just add these information to the comments.